### DISCUSSION TITLE
Reclaim areas of deleted islands

### DISCUSSION THREAD

**Discussion by wlm1455925277** [BG-Software-LLC/SuperiorSkyblock2#2075] [Pending Feature Requests] [answered]
I found that when the player deletes the island, a new island will be created at the coordinates of the next island. If a player creates a new island after the player deletes the island, then the new island will be created at the coordinates of the deleted island! This can obviously improve performance and reduce map waste and memory usage~

**Comment by Lemvs** [comment, marked as answer]
Duplicate of https://github.com/BG-Software-LLC/SuperiorSkyblock2/discussions/211

@OmerBenGera you can close it.
